opencti-platform/opencti-front/src/private/components/analyses/malware_analyses/Root.tsx

```diff
@@ -76,6 +77,11 @@ const RootMalwareAnalysis = () => {
   const location = useLocation();
   const { t_i18n } = useFormatter();
   useSubscription(subConfig);
+  const getCurrentTab = (malwareAnalysis) => {
```

Comment:
is it possible to create a generic method for all containers ? that we could call in each component with location, entity_type_path (malware_analyses, groupings, reports, ...), entity_id.